README.md

```diff
@@ -136,21 +152,23 @@ liboqs-python can be imported into Python programs with
 import oqs
 ```
 
-liboqs-python defines two main classes: `KeyEncapsulation` and `Signature`, providing post-quantum key encapsulation and
-signture mechanisms, respectively. Each must be instantiated with a string identifying one of mechanisms supported by
-liboqs; these can be enumerated using the `get_enabled_KEM_mechanisms()` and `get_enabled_sig_mechanisms()` functions.
-The files in `examples/` demonstrate the wrapper's API. Support for alternative RNGs is provided via
-the `randombytes_*()`
-functions.
+liboqs-python defines two main classes: `KeyEncapsulation` and `Signature`,
+providing post-quantum key encapsulation and signture mechanisms, respectively.
+Each must be instantiated with a string identifying one of mechanisms supported
+by liboqs; these can be enumerated using the `get_enabled_KEM_mechanisms()` and
+`get_enabled_sig_mechanisms()` functions. The files in `examples/` demonstrate
```

Comment:
Different capitalization KEM/sig?

Reply:
For some "historical" reason I used KEM and "sig" across the wrappers, so it'll break the API if I rename it. Perhaps in a future version.